requirements-dev.txt

```diff
@@ -1 +1,3 @@
-bumpversion
```

Comment:
I think `requirements-dev.txt` should be deleted actually.  Hasn't it been supplanted with `pip install -e .[test,lint]`?

Reply:
And where's your cute animal??

Reply:
Neat, thanks. Done in c12534c5e9

Reply:
I was in a rush and left my cute animal at home! Added